includes/Admin/Crop_Script.php

```diff
@@ -0,0 +1,115 @@
+<?php
+/**
+ * Class Crop_Script.
+ *
+ * @package   Google\Web_Stories
+ * @copyright 2020 Google LLC
+ * @license   https://www.apache.org/licenses/LICENSE-2.0 Apache License 2.0
+ * @link      https://github.com/google/web-stories-wp
+ */
+
+/**
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+namespace Google\Web_Stories\Admin;
+
+use Google\Web_Stories\Assets;
+use Google\Web_Stories\Infrastructure\Conditional;
+use Google\Web_Stories\Infrastructure\Registerable;
+use Google\Web_Stories\Infrastructure\Service;
+use Google\Web_Stories\Traits\Screen;
+
+/**
+ * Class Crop Script
+ *
+ * @package Google\Web_Stories
+ */
+class Crop_Script implements Conditional, Service, Registerable {
```

Comment:
@swissspidy Please help me with a better name here. Couldn't think of anything.

Reply:
`ImgAreaSelect_Patch`?